WorkflowSwiftUI/Sources/ActionModel.swift

```diff
@@ -5,6 +5,11 @@
 public struct ActionModel<State: ObservableState, Action>: ObservableModel, SingleActionModel {
     public let accessor: StateAccessor<State>
     public let sendAction: (Action) -> Void
+
+    public init(accessor: StateAccessor<State>, sendAction: @escaping (Action) -> Void) {
```

Comment:
are there any reasons we'd want to restrict clients' ability to create these types in general?

Reply:
To reduce the chance of folks using it wrong, mainly. You can't create something like a `RenderContext` either. The type's docs already hint at this, and I can copy it to the init as well.

Reply:
to be clear – you think the convenience of exposing the public constructor outweighs the chance of misuse?